views/notice-expired.html

```diff
@@ -20,7 +20,7 @@ <h1 class="govuk-heading-xl">
           to find out more about restoring a company name to the register.</p>
 
       <p>If this is the wrong company, start the
-        <a href="/company-number">Apply to object to a company being struck off</a>
+        <a href="/strike-off-objections">Apply to object to a company being struck off</a>
```

Comment:
This was a broken link and wasn't going back to the start of the journey, as required in story. Fixed it here and on the new screen.